src/metabase/api/embed.clj

```diff
@@ -343,7 +348,7 @@
 
   Returns a `StreamingResponse`."
   {:style/indent 1}
-  [token dashcard-id card-id export-format query-params
+  [token dashcard-id card-id export-format query-params qp-runner
```

Comment:
Can we make `qp-runner` a keyword param instead of adding another positional param to a bunch of functions? 5 positional params is enough